catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -223,6 +236,8 @@ public Metacard transform(InputStream input, String id)
             }
 
             String metacardContentType = metacard.getContentTypeName();
+            metacard.setAttribute(new AttributeImpl(Core.DATATYPE, getDatatype(metacardContentType)));
```

Comment:
Refactored to map some specific and commonly used mime types more appropriately.  My concern with creating a PreIngestPlugin is that we would still have to modify all of the transformers  to set the mime type in order for the plugin to appropriately map it to a datatype.  It seems more appropriate to cut out the middleman, and just set the datatype in the transformer directly.